lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py

```diff
@@ -0,0 +1,1509 @@
+            eg_strategy.signals = eg_signals
+
+    eg.strategy = eg_strategy
+
+
+def expand_multai(eg, module):
+    multai_load_balancers = module.params.get('multai_load_balancers')
+
+    eg_multai = expand_fields(multai_fields, module.params, 'Multai')
+
+    if multai_load_balancers is not None:
+        eg_multai_load_balancers = expand_list(multai_load_balancers, multai_lb_fields, 'MultaiLoadBalancer')
+
+        if eg_multai_load_balancers.__sizeof__() > 0:
+            eg_multai.balancers = eg_multai_load_balancers
+            eg.multai = eg_multai
+
+
+def expand_scheduled_tasks(eg, module):
+    scheduled_tasks = module.params.get('scheduled_tasks')
+
+    if scheduled_tasks is not None:
+        eg_scheduling = spotinst.aws_elastigroup.Scheduling()
+
+        eg_tasks = expand_list(scheduled_tasks, scheduled_task_fields, 'ScheduledTask')
+
+        if eg_tasks.__sizeof__() > 0:
+            eg_scheduling.tasks = eg_tasks
+            eg.scheduling = eg_scheduling
+
+
+def expand_load_balancers(eg_launchspec, load_balancers, target_group_arns):
+    if load_balancers is not None or target_group_arns is not None:
+        eg_load_balancers_config = spotinst.aws_elastigroup.LoadBalancersConfig()
+        eg_total_lbs = []
+
+        if load_balancers is not None:
+            for elb_name in load_balancers:
+                eg_elb = spotinst.aws_elastigroup.LoadBalancer()
+                if elb_name is not None:
+                    eg_elb.name = elb_name
+                    eg_elb.type = 'CLASSIC'
+                    eg_total_lbs.append(eg_elb)
+
+        if target_group_arns is not None:
+            for target_arn in target_group_arns:
+                eg_elb = spotinst.aws_elastigroup.LoadBalancer()
+                if target_arn is not None:
+                    eg_elb.arn = target_arn
+                    eg_elb.type = 'TARGET_GROUP'
+                    eg_total_lbs.append(eg_elb)
+
+        if eg_total_lbs.__sizeof__() > 0:
+            eg_load_balancers_config.load_balancers = eg_total_lbs
+            eg_launchspec.load_balancers_config = eg_load_balancers_config
+
+
+def expand_tags(eg_launchspec, tags):
+    if tags is not None:
+        eg_tags = []
+
+        for tag in tags:
+            eg_tag = spotinst.aws_elastigroup.Tag()
+            if tag.keys():
+                eg_tag.tag_key = tag.keys()[0]
+            if tag.values():
+                eg_tag.tag_value = tag.values()[0]
+
+            eg_tags.append(eg_tag)
+
+        if eg_tags.__sizeof__() > 0:
+            eg_launchspec.tags = eg_tags
+
+
+def expand_block_device_mappings(eg_launchspec, bdms):
+    if bdms is not None:
+        eg_bdms = []
+
+        for bdm in bdms:
+            eg_bdm = expand_fields(bdm_fields, bdm, 'BlockDeviceMapping')
+
+            if bdm.get('ebs') is not None:
+                eg_bdm.ebs = expand_fields(ebs_fields, bdm.get('ebs'), 'EBS')
+
+            eg_bdms.append(eg_bdm)
+
+        if eg_bdms.__sizeof__() > 0:
```

Comment:
Please use `if len(eg_bdms) > 0` instead of directly calling `__sizeof__`

Reply:
Changed all sizeof to len()